connectors/es/client.py

```diff
@@ -162,36 +159,40 @@ async def wait(self):
         while time.time() - start < self.max_wait_duration:
             if not self._keep_waiting:
                 await self.close()
-                return False
+                return None
 
             logger.info(
                 f"Waiting for Elasticsearch at {self.configured_host} (so far: {int(time.time() - start)} secs)"
             )
             logger.debug(
                 f"Seed node configuration: {self.client.transport.node_pool._seed_nodes}"
             )
-            if await self.ping():
-                return True
+            response = await self.ping()
+            if response is not None:
+                return response
             await self._sleeps.sleep(backoff)
             backoff *= self.backoff_multiplier
 
         await self.close()
-        return False
+        return None
 
     async def ping(self):
         try:
-            await self.client.info()
+            response = await self.client.info()
+            logger.info(
+                f"Connected to Elasticsearch server with version '{response['version']['number']}' and build flavor '{response['version']['build_flavor']}'"
+            )
```

Comment:
Is this going to be logged frequently? If so, we may want to make it `debug`

Reply:
It should only happen on startup but I made it debug anyway, as the version compatibility check will output something afterwards anyway.